appveyor.yml

```diff
@@ -54,21 +54,27 @@ install:
 
     # Cywing's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
     - cmd: rmdir C:\cygwin /s /q
+    # Appveyor's git for windows contains cygwin and breaks this build
+    - cmd: rmdir "C:\Program Files\Git" /s /q
     - appveyor DownloadFile "https://raw.githubusercontent.com/pelson/Obvious-CI/master/bootstrap-obvious-ci-and-miniconda.py"
     - cmd: python bootstrap-obvious-ci-and-miniconda.py %CONDA_INSTALL_LOCN% %TARGET_ARCH% %CONDA_PY:~0,1% --without-obvci
     - cmd: set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts;%PATH%
     - cmd: set PYTHONUNBUFFERED=1
 
     - cmd: conda install -c http://conda.binstar.org/pelson/channel/development --yes --quiet obvious-ci
     - cmd: conda config --add channels http://conda.binstar.org/conda-forge
+    - cmd: conda config --add channels msys2
     - cmd: conda info
-    - cmd: conda install -n root --quiet --yes conda-build anaconda-client jinja2 setuptools
+    - cmd: conda install -n root --quiet --yes anaconda-client jinja2 setuptools m2-git m2-base
```

Comment:
Yeah, `m2-*` will always only be build dependencies since they're the POSIX part of MSYS2.

If you start using `m2w64-*` packages then they'll often be run dependencies since they're the mingw-w64 libraries.